src/app/components/ElectronicDeliveryForm/ElectronicDeliveryForm.jsx

```diff
@@ -1,13 +1,12 @@
-import React from 'react';
 import PropTypes from 'prop-types';
```

Comment:
Committed Prettied file. No Functional Changes were done to this file. Soley format.